### ISSUE TITLE
WCAG 2.1 Compliance

### ISSUE THREAD

**Issue by lmrodriguezr** [seq-code/registry#117]
The Registry should be WCAG 2.1 compliant. Evaluation tools such as [accessScan](https://www.accessiway.com/scan-your-website) can help identify accessibility issues.
